docs/data/joy/getting-started/tutorial/tutorial.md

```diff
@@ -0,0 +1,232 @@
+# Tutorial
+
+<p class="description">This tutorial gives you an overview of how to use Joy components by building a log in page.</p>
+
+## What you'll learn?
+
+1. Importing components.
+2. Building a log in page using Joy components.
+3. What global variants look like.
+4. Toggling dark mode
```

Comment:
I feel that `Understanding global variants` might be a bit misleading here. In this tutorial, we won't explain in detail the global variant, we just show the developer what it looks like. For sure they will be curious about it and that's the purpose of this tutorial so that they find out more at the end of the tutorial.